docs/using.rst

```diff
@@ -44,17 +24,62 @@ above method instead.
 
 .. _prerequisites:
 
-Prerequisites
-=============
+Installation and Usage
+======================
+
+To install and run the client you just need to type:
+
+.. code-block:: shell
+
+   ./letsencrypt-auto
+
+.. warning:: Please do **not** use ``python setup.py install``.  That mode of
+             operation might corrupt your operating system and is **not supported**
+             by the Let's Encrypt team!
+
+The ``letsencrypt`` commandline tool has a builtin help:
+
+.. code-block:: shell
+
+   ./letsencrypt-auto --help
+
+
+Configuration file
+------------------
+
+It is possible to specify configuration file with
+``letsencrypt --config cli.ini`` (or shorter ``-c cli.ini``). For
+instance, if you are a contributor, you might find the following
+handy:
+
+.. include:: ../examples/dev-cli.ini
+   :code: ini
+
+By default, the following locations are searched:
+
+- ``/etc/letsencrypt/cli.ini``
+- ``$XDG_CONFIG_HOME/letsencrypt/cli.ini`` (or
+  ``~/.config/letsencrypt/cli.ini`` if ``$XDG_CONFIG_HOME`` is not
+  set).
+
+.. keep it up to date with constants.py
+
+
+.. _Augeas: http://augeas.net/
+.. _Virtualenv: https://virtualenv.pypa.io
+
+Notes on OS depedencies
+=======================
 
-The demo code is supported and known to work on **Ubuntu and
-Debian**. Therefore, prerequisites for other platforms listed below
-are provided mainly for the :ref:`developers <hacking>` reference.
+OS level dependencies are managed by scripts in ``bootstrap``.  Some notes
+are provided here mainly for the :ref:`developers <hacking>` reference.
 
 In general:
 
 * ``sudo`` is required as a suggested way of running privileged process
 * `Augeas`_ is required for the Python bindings
+* ``virtualenv`` and ``pip`` are used for managing other python library
```

Comment:
Listing `pip` is likely to be very confusing: c.f. https://github.com/letsencrypt/letsencrypt/pull/963#issuecomment-148278310. `pip` is installed automatically on `virtualenv` invocation - we don't use global `pip`.